apidocs/index.rst

```diff
@@ -0,0 +1,20 @@
+.. Surfactant Documentation documentation master file, created by
+   sphinx-quickstart on Mon Oct 28 15:43:14 2024.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Surfactant Documentation
+======================================
+
+Add your content using ``reStructuredText`` syntax. See the
+`reStructuredText <https://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html>`_
+documentation for details.
+
+
+.. toctree::
+   :maxdepth: 4
+   :caption: Contents:
+
+   modules
```

Comment:
This can be added to the existing docs/index page (rename modules to whatever the new file name is (eg api_reference)